src/main/java/edu/kit/hci/soli/controller/MainController.java

```diff
@@ -1,37 +1,36 @@
 package edu.kit.hci.soli.controller;
 
 import edu.kit.hci.soli.domain.LoginStateModel;
-import edu.kit.hci.soli.repository.VisitsRepository;
-import jakarta.servlet.http.HttpServletResponse;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
+import edu.kit.hci.soli.repository.RoomRepository;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.security.core.annotation.AuthenticationPrincipal;
-import org.springframework.security.oauth2.core.oidc.user.OidcUser;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import java.security.Principal;
-
 @Controller("/")
+@Slf4j
 public class MainController {
 
-    private final Logger log = LoggerFactory.getLogger(MainController.class);
-
     @Value("${spring.profiles.active}")
     private String profile;
 
+    private final RoomRepository roomRepository;
+
+    public MainController(RoomRepository roomRepository) {
+        this.roomRepository = roomRepository;
+    }
+
     @ResponseBody
     @GetMapping("/profile")
     public String getProfile() {
         return profile;
     }
 
     @GetMapping("/")
-    public String index(@ModelAttribute("login") LoginStateModel login) {
-        if (!login.name.equals("")) log.info("Received request from {}", login.name);
+    public String index(Model model, @ModelAttribute("login") LoginStateModel login) {
+        if (!login.name().isEmpty()) log.info("Received request from {}", login.name());
+        model.addAttribute("room", roomRepository.findById(1L).orElseThrow()); //TODO allow other rooms
```

Comment:
We need a reference to the room to get the ID, so I decided to inject the room this way.
We could pass the ID through the URL and add a redirect from `/`, though.